source/jsonnet/england-wales/lib/placeholders.libsonnet

```diff
@@ -66,4 +66,62 @@
       },
     }],
   },
+  ordinal: {
+    placeholder: 'ordinal',
```

Comment:
This should be more descriptive.

Reply:
ordinalHoushold?

Reply:
Changed it to `household_ordinality`, thanks!